build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1961,9 +1961,30 @@ message Compressor {
 
     // Brotli compression.
     BROTLI = 3;
+
+    // Zstandard compression with dictionary.
+    //
```

Comment:
Should we provide advice on which dictionary to use for a given blob, and which not to use a dictionary for at all? eg maybe large blobs over a certain size aren't worth using a dictionary for. And maybe we could use file type specific dictionaries?

Reply:
We could discuss the mixed usage of different compressors separately in another change, but at least for now, I would prefer to treat this the same way as treating `ZSTD vs Brotli`. Let's leave the decision to the actual implementations to decide.

I think once this is implemented by a set of clients and servers, we will have a better idea regarding performance implications, as well as whether a mixed use of compressors is preferred in certain scenarios.